bindings/ocaml/lib/opendal.ml

```diff
@@ -19,4 +19,4 @@
 
 module Operator = Operator
 
-type operator = Opendalinner.Operator.operator
+type operator = Opendal_core.Operator.operator
```

Comment:
Oh, can we use the consist style? For example: `OpendalCore` or `opendal_core`.

Reply:
The first letter of Module must be capitalized, We can use `OpendalCore`.